dynawo/sources/Models/Modelica/Dynawo/Electrical/StaticVarCompensators/SVarCStandard.mo

```diff
@@ -23,21 +23,26 @@ model SVarCStandard "Standard static var compensator model"
   extends BaseControls.Parameters.Params_Limitations;
   extends BaseControls.Parameters.Params_CalculBG;
   extends BaseControls.Parameters.Params_ModeHandling;
-
-  Modelica.Blocks.Interfaces.RealInput URef(start = URef0) "Voltage reference for the regulation in kV" annotation(
-    Placement(visible = true, transformation(origin = {-200, 28}, extent = {{-20, -20}, {20, 20}}, rotation = 0), iconTransformation(origin = {-115, 85}, extent = {{-15, -15}, {15, 15}}, rotation = 0)));
-
-  Modelica.Blocks.Interfaces.BooleanInput selectModeAuto(start = selectModeAuto0) "Wheter the static var compensator is in automatic configuration" annotation(
-    Placement(visible = true, transformation(origin = {-200, 56}, extent = {{-20, -20}, {20, 20}}, rotation = 0), iconTransformation(origin = {-115, 1}, extent = {{-15, -15}, {15, 15}}, rotation = 0)));
-  Modelica.Blocks.Interfaces.IntegerInput setModeManual(start = setModeManual0) "Mode selected when in manual configuration" annotation(
-    Placement(visible = true, transformation(origin = {-200, 86}, extent = {{-20, -20}, {20, 20}}, rotation = 0), iconTransformation(origin = {-115, -79}, extent = {{-15, -15}, {15, 15}}, rotation = 0)));
+  extends BaseControls.Parameters.Params_BlockingFunction;
+  parameter Types.VoltageModule UNom "Static var compensator nominal voltage in kV";
+  final parameter Types.VoltageModule UThresholdUpPu =  UThresholdUp / UNom;
```

Comment:
I would like to but I need UNom in both cases. So if I have UNom in Params_ModeHandling and Params_BlockingFunction, I will have an error in Modelica when extending both in the same model. Do you know some way to avoid this ?

Reply:
We can create a Param_Nominal with UNom and SNom (remove it from Param_Regulation) and extends it several times?

Reply:
Then in the Params_Blocking and Params_ModeHandling, I will have the final parameters but no UNom, which will give an error no ?
If I extend the Param_Nominal in the two Params, when using both I think I will have a double declaration.

Reply:
Ah you're right! I have no other idea right away...